src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftFeatures.kt

```diff
@@ -24,6 +24,7 @@ public open class RedshiftFeatures : SqlFeatures.Defensive() {
         Rel.Op.Exclude::class.java,
         Rel.Op.Exclude.Item::class.java,
         Rel.Op.Exclude.Step.StructField::class.java,
+        Rel.Op.Union::class.java,
```

Comment:
Only enabled `UNION` for now. This PR does support transpiling `EXCEPT` and `INTERSECT` to the other dialects though. Perhaps we could enable it?

Reply:
Just UNION is fine for now.